backend/organization/models/organization.py

```diff
@@ -91,6 +91,31 @@ class Organization(models.Model):
         blank=True
     )
 
+    location = models.CharField(
+        help_text="Organization location",
+        verbose_name="Location",
+        max_length=2048,
+        null=True,
+        blank=True
+    )
+
+
+    latitude = models.CharField(
+        help_text="Latitude of organization's location",
+        verbose_name="Latitude",
+        max_length=512,
+        null=True,
+        blank=True
+    )
+
+    longitude = models.CharField(
+        help_text="Longitude of organization's location",
+        verbose_name="Longitude",
+        max_length=512,
+        null=True,
+        blank=True
+    )
+
     short_description = models.TextField(
```

Comment:
The migration has `loc` as a foreign key, but the Organization model is missing.

Reply:
backend/organization/migrations/0060_auto_20210121_1040.py